Kernel/Output/HTML/FilterElementPost/ITSMIncidentProblemManagement.pm

```diff
@@ -58,6 +58,23 @@ sub Run {
 
         # get ticket id
         my $TicketID = $ParamObject->GetParam( Param => 'TicketID' );
+        my $TicketNr = $ParamObject->GetParam( Param => 'TicketNumber' );
+
+        # get ticket id in case necessary
+        if ( ! $TicketID ) {
+            $TicketID = $TicketObject->TicketIDLookup(
+                TicketNumber => $TicketNr,
+            );
+        }
+
+        if ( ! $TicketID ) {
+            $Kernel::OM->Get('Kernel::System::Log')->Log(
+                Priority => 'error',
+                Message  => "Need TicketID or TicketNumber!",
+            );
+            return;
+        }
+
```

Comment:
```suggestion
        # get ticket id
        my $TicketID     = $ParamObject->GetParam( Param => 'TicketID' );
        my $TicketNumber = $ParamObject->GetParam( Param => 'TicketNumber' );

        # get ticket id in case necessary
        if ( !$TicketID ) {
            $TicketID = $TicketObject->TicketIDLookup(
                TicketNumber => $TicketNumber,
            );
        }

        if ( !$TicketID ) {
            $Kernel::OM->Get('Kernel::System::Log')->Log(
                Priority => 'error',
                Message  => "Need TicketID or TicketNumber!",
            );
            return;
        }
        
```